server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraMailboxModule.java

```diff
@@ -120,6 +121,7 @@ protected void configure() {
         bind(CassandraUidProvider.class).in(Scopes.SINGLETON);
         bind(NoMailboxPathLocker.class).in(Scopes.SINGLETON);
 
+        bind(SubscriptionMapperFactory.class).to(CassandraMailboxSessionMapperFactory.class);
```

Comment:
alignment

Reply:
What do you mean?

Reply:
Nothing sorry, it's the view in github that maked this line appears not indented like the others.